projects/web-app/src/views/meta.view.vue

```diff
@@ -28,19 +28,22 @@
 </template>
 
 <script>
-import { computed } from 'vue';
+import { computed, onMounted, ref } from 'vue';
 import useHead from '@/composables/use-head';
-import { findCharset, findMetaContent, findAttrs } from '@shared/lib/find-meta';
-import schema from '@shared/lib/schemas/app-meta-schema';
+import { findCharset, findMetaContent } from '@shared/lib/find-meta';
+import validateData from '@shared/lib/validate-data';
+import info from '@shared/lib/schemas/app-meta-schema';
```

Comment:
Can we rename the `app-meta-schema` file and clean it up? It's a little confusing now, and has a lot of redundant info.

Reply:
You could also consider having one file, `app-meta-schema`, which holds two objects. One for the validator, and one pure meta for tooltips. So you'd merge `app-meta-schema` and `app-meta-validation`.

Reply:
Done.